### DISCUSSION TITLE
Request Body Value as File

### DISCUSSION THREAD

**Discussion by AkkafaOguz** [usebruno/bruno#1018] [Q&A]
Is there a possibility to send file (PNG, JPG or PDF) in the request body like we can do at Postman? (Not uploading a file)

**Comment by R-0-m-a** [comment]
Bruno v1.33.1
Currently, there is still no option to upload a file as the request body in the Bruno app. Is there any plan to add this feature?

**Reply by rreyn-bruno** [reply]
Hi @R-0-m-a, unless I'm misunderstanding, you do have the ability to send form data as the request body similar to Postman. 
![image](https://github.com/user-attachments/assets/51b7bcf0-d32c-489d-99ef-4f612611bca3)

Could you give me more detail on what you're trying to accomplish?

**Reply by R-0-m-a** [reply]
Hi @TheRealRyanReynolds, sure.
I often have to work with a Content-Type like application/pdf. Postman allows uploading a PDF file as the body. But in Bruno, the body can only be JSON, XML, TEXT, or SPARQL.
<img width="504" alt="SCR-20241030-tfkq-2" src="https://github.com/user-attachments/assets/0e68c988-75ed-456c-81ab-4405dbfd0036">
<img width="521" alt="SCR-20241030-tgcf-2" src="https://github.com/user-attachments/assets/640699c2-9f72-4d8b-aef8-e97e8234758d">

**Reply by R-0-m-a** [reply]
I have found an open issue - https://github.com/usebruno/bruno/issues/1992
